source/logHandler.py

```diff
@@ -317,6 +317,12 @@ def initialize(shouldDoRemoteLogging=False):
 			except (IOError, WindowsError):
 				pass # Probably log does not exist, don't care.
 			logHandler = FileHandler(globalVars.appArgs.logFileName, mode="w",encoding="utf-8")
+			logLevel = globalVars.appArgs.logLevel
+			if globalVars.appArgs.debugLogging:

```

Comment:
Shouldn't there be a clause here that checks `globalVars.appArgs.noLogging` and sets the level to `log.OFF`?

Reply:
Nope, `globalVars.appArgs.noLogging does not apply in the `else` clause we are in.

As a reminder:
 - `--debug-logging` takes precedence over `--secure`, `--log-level` and `--no-logging`
 - `--log-level` takes precedence over `--secure` and `--no-logging`

I am not sure about the motivations behind the historical decision of not making `--debug-logging`, `--log-level` and `--no-logging` mutually exclusive, though.